Monika After Story/game/script-stories.rpy

```diff
@@ -1015,3 +1038,295 @@ label mas_scary_story_revenant:
+    m 2dsc "A few days later, the Admiralty inquired the couple about a missing passenger from the wreckage that turned out to be none other than their daughter."
+    m 3dsd "Devastated and ashamed, the parents walled up her remains in a secret room and moved away, never to return."
+    call mas_scary_story_cleanup
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_story_database,
+            eventlabel="mas_scary_story_flowered_lantern",
+            category=[store.mas_stories.TYPE_SCARY],
+            prompt="The Flowered Lantern",
+            unlocked=False
+        ),
+        code="STY"
+    )
+    
+label mas_scary_story_flowered_lantern:
+    call mas_scary_story_setup
+    $ _story = mas_getEV('mas_scary_story_flowered_lantern')
+    if _story is not None and _story.shown_count == 0:
+        m 3eub "Before we start, I need to tell you that my next story is going to be a bit long."
+        m 3eua "So, I'll split it in parts."
+        m "Once I finish this part I'll ask you if you want to continue it or not."
+        m 1eub "If you say no, you can ask me later to tell you the next part, so don't worry about it."
+        m 4hua "Alright, let's begin now."
+    m 4eua "There was once a beautiful, young maiden named Tsuyu, whose father was a high-ranking samurai."
+    m 4eud "Tsuyu's mother had been dead and her father eventually remarried."
+    m 2euc "Although it became obvious to Tsuyu's father that she and her step mother couldn't get along."
+    m 2esa "Wanting to ensure his only daughter's happiness, he had a luxurious house built for her, away from them and had her move into it."
+    m "One day, the family physician went to Tsuyu's residence on a routine visit with a young samurai named Hagiwara, who was very handsome."
+    m 4eub "Tsuyu and Hagiwara fell in love the moment they set eyes on each other."
+    m 4esc "Unbeknownst to the doctor, the two pledged themselves to each other for life and before the two left."
+    m 4dsd "Tsuyu whispered to Hagiwara that she would surely die if he didn't return to see her."
+    m 2esc "Hagiwara didn't forget her words, but etiquette forbade him to make a call to visit a maiden alone, so he had to wait for the doctor to ask him to join him on another visit."
+    m 2dsd "The doctor, however, had perceived his sudden affection for Tsuyu."
+    m 4ekc "Tsuyu's father was known to behead those who angered him, and fearing that he would hold him responsible for introducing the two, he avoided Hagiwara."
+    m 2rkc "Months passed and Tsuyu, feeling scorned that Hagiwara had abandoned her, passed away."
+    m 1ekc "Not long afterwards, the doctor ran into Hagiwara, informing him of Tsuyu's death."
+    m 1dsd "Hagiwara was deeply saddened and mourned greatly over her, saying prayers and burning incense for her."
+    $ mas_stories.unlock_pooled_story("mas_scary_story_flowered_lantern_2")

```

Comment:
is this supposed to make the story appear in the list? The `seen_event` condition in the menu generation code means unlocking the story won't actually make it appear in the list.